src/config/base.cpp

```diff
@@ -132,6 +169,25 @@ bool ConfigBase::needs_push() const {
     return !is_clean();
 }
 
+// Tries to compresses the message; if the compressed version (including the 'z' prefix tag) is
+// smaller than the source message then we modify `msg` to contain the 'z'-prefixed compressed
+// message and return true; otherwise we leave msg alone and return false.
```

Comment:
you're not returning anything here

Reply:
Thanks!  An intermediate version had a bool return here but I removed it and forgot to update the comment!

Apparently I also forgot to add the test code.